azure/packages/azure-local-service/package.json

```diff
@@ -23,8 +23,11 @@
     "clean": "rimraf dist lib *.tsbuildinfo *.build.log",
     "eslint": "eslint --format stylish src",
     "eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
+    "format": "npm run prettier:fix",
     "lint": "npm run eslint",
     "lint:fix": "npm run eslint:fix",
+    "prettier": "prettier --check . --ignore-path ../../../.prettierignore",
```

Comment:
This relative path is outside of the repo, maybe bad copy/paste?

Reply:
It's outside the azure mono-repo, but still in the fluidframework repo. There is an ignore file at the root of our repo. We may want to the ignore files into each mono-repo long-term, though, if we want to properly support each as its own isolated workspace.

Reply:
Ah right - though typically configs and such in the root are implied to be specific to the client packages.